src/main/java/net/dries007/tfc/objects/te/TEAnvilTFC.java

```diff
@@ -335,7 +335,7 @@ public boolean attemptWelding(EntityPlayer player)
             if (fluxStack.isEmpty())
             {
                 // No flux
-                player.sendMessage(new TextComponentTranslation(MOD_ID + ".tooltip.anvil_no_flux"));
+                player.sendStatusMessage(new TextComponentTranslation(MOD_ID + ".tooltip.anvil_no_flux"), ConfigTFC.Client.TOOLTIP.anvilWeldOutputToActionBar);
```

Comment:
So as I learned as per #1647, this will actually not work on a dedicated server, as the client config will be referenced from the server, ergo not really serving the intended function. So if this is to be desired, it needs to be implemented via packet and the config checked on client.

Reply:
We can probably make a general purpose packet for "send this data to client so that they display a message in the appropriate place as per local configs" that can be used across the board...

... or maybe even intercept these messages on the client side and override where they are meant to be displayed?

Reply:
So while I was doing some testing, turns out the tile entity has the welding method called by both client and server, but doesn't receive the world along with the player meaning you can't check if its the server so you can send the packet via SimpleNetworkWrapper. 

https://github.com/TerraFirmaCraft/TerraFirmaCraft/blob/1.12.x/src/main/java/net/dries007/tfc/objects/blocks/metal/BlockAnvilTFC.java#L213

Reply:
This also explains why welding error messages are received twice

Reply:
`player.world`?

Reply:
oh duh, I forgot that. Is there any reason we need welding to occur on both sides? (besides sound + particles)

Reply:
It really shouldn't be, probably the cause of #1075